oauth2client/service_account.py

```diff
@@ -461,3 +465,94 @@ def create_delegated(self, sub):
         result._private_key_pkcs12 = self._private_key_pkcs12
         result._private_key_password = self._private_key_password
         return result
+
+
+def _datetime_to_secs(utc_time):
+   # time_delta.total_seconds() not supported in Python 2.6
+   epoch = datetime.datetime(1970, 1, 1)
+   time_delta = utc_time - epoch
+   return time_delta.days * 86400 + time_delta.seconds
+
+
+class JWTAccessCredentials(GoogleCredentials):
```

Comment:
You've chosen to use inheritance, so welcome to the inheritance questionnaire! :-)

* Why should instances of this class be-a `GoogleCredentials` rather than have-a `GoogleCredentials`?
* Does this class conform to the [Liskov Substitution Principle](https://en.wikipedia.org/wiki/Liskov_substitution_principle)?
  * Are there places where a `GoogleCredentials` is requested and at those places is a `JWTAccessCredentials` acceptable?
  * Are there places where a `OAuth2Credentials` is requested and at those places is a `JWTAccessCredentials` acceptable?
  * Are there places where a `Credentials` is requested and at those places is a `JWTAccessCredentials` acceptable?
* Within the body of this class, which code elements override inherited concrete elements?
* Within the body of this class, which code elements implement inherited abstract elements?
* This class has seven public instance methods and one public class method. How many of those are intended to be used by users of the `oauth2client` library?